dnf/cli/output.py

```diff
@@ -20,6 +20,7 @@
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import unicode_literals
+from dnf.cli.format import format_date, format_datetime
```

Comment:
maybe
import dnf.cli.format as format 
at use format.format_date & format.format_datetime, will show the origin of the function without the the very long dnf.cli.format. prefix